cmd/zqd/listen/command.go

```diff
@@ -23,28 +26,70 @@ The listen command launches a process to listen on the provided interface and
 	New: New,
 }
 
+// Example configfile
+// loggers:
+// - path: ./data/access.log
+//   name: "http.access"
+//   level: info
+//   mode: truncate
+
```

Comment:
Maybe have this adjacent to loadConfigFile so the two are more likely to stay in sync.

Reply:
Good idea.